regolith/helpers/a_manurevhelper.py

```diff
@@ -0,0 +1,125 @@
+"""Builder for Current and Pending Reports."""
+import datetime as dt
+import sys
+import time
+from argparse import RawTextHelpFormatter
+
+import nameparser
+
+from regolith.helpers.basehelper import SoutHelperBase, DbHelperBase
+from regolith.dates import month_to_int, month_to_str_int
+from regolith.fsclient import _id_key
+from regolith.sorters import position_key
+from regolith.tools import (
+    all_docs_from_collection,
+    filter_grants,
+    fuzzy_retrieval,
+)
+
+ALLOWED_TYPES = ["nsf", "doe", "other"]
```

Comment:
is this correct?

Reply:
"type" isn't really required for this so maybe I should just delete this line?